api/server/router/network/network_routes.go

```diff
@@ -198,6 +210,15 @@ func (n *networkRouter) buildNetworkResource(nw libnetwork.Network) *types.Netwo
 		r.Peers = buildPeerInfoResources(peers)
 	}
 
+	return r
+}
+
+func (n *networkRouter) buildNetworkDetailResources(nw libnetwork.Network) *types.NetworkResource {
```

Comment:
not tryin to bikeshed but i find the function name a bit weird. How about `buildDetailedNetworkResource` since `NetworkResource` is the normal one.